pynest/nest/tests/test_split.py

```diff
@@ -0,0 +1,81 @@
+# -*- coding: utf-8 -*-
```

Comment:
Can you please rename this file to something like `test_split_simulation.py`? I think the current name is a bit too general.